File: Modul-4/Python/PRAK402-2310817110006-Adrian_Bintang_Saputera.py
batas = int(input("Input\n"))
print("Output")

for i in range(1, batas+1, 2):
    print(i, end=" ")

print()

if batas % 2 == 0:
    for i in range(batas, 1, -2):
        print(i, end=" ")
else:
    for i in range(batas-1, 1, -2):
        print(i, end=" ")

File: Modul-4/C/PRAK402-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int batas, i;

  printf("Input\n");
  scanf("%d", &batas);

  printf("Output\n");

  for (i = 1; i <= batas; i += 2)
  {
    printf("%d ", i);
  }

  printf("\n");

  if (batas % 2 == 0)
  {
    for (i = batas; i >= 2; i -= 2)
    {
      printf("%d ", i);
    }
  }
  else
  {
    for (i = batas - 1; i >= 2; i -= 2)
    {
      printf("%d ", i);
    }
  }

  return 0;
}